### ISSUE TITLE
Hello

### ISSUE THREAD

**Issue by sharpaper** [rdfhdt/hdt-mr#8]
Any dev on the list? Is this project still active? Or dead?

**Comment by akuckartz** [comment]
:+1:

**Comment by sharpaper** [comment]
@akuckartz are you a hdt-mr dev?

**Comment by akuckartz** [comment]
@sharpaper No, my :+1: means only that I like your question. The mobile Github interface unfortunately does not allow to :+1: issues or comments.

**Comment by jm-gimenez-garcia** [comment]
I was the main developer of hdt-mr, but unfortunately I have not much time or suitable resources to maintain it at the moment (I hope that changes in the future). I can try to help with any question, though.

**Comment by tangina-sultana** [comment]
Can you help me to compile the HDT-MR code by providing the installation manual. Actually the readme file does not describe it clearly. I am interested to run this code. Please help me
